client/rest-client/src/main/java/org/infinispan/client/rest/impl/okhttp/RestClientOkHttp.java

```diff
@@ -190,6 +190,14 @@ public CompletionStage<RestResponse> caches() {
       return execute(baseURL, configuration.contextPath(), "v2", "caches");
    }
 
+   @Override
+   public CompletionStage<RestResponse> caches(String roleName) {
```

Comment:
The name of this method should be a bit more descriptive, such as `cachesByRole`